js/mega-menu.js

```diff
@@ -0,0 +1,68 @@
+// Toggles the mega menu
+$('#mega-menu-pull-down, #mega-menu-mobile').on('click', function () {
+    $('#nav').slideToggle('fast');
+});
+
+// Displays the promotional image
+
+$("ul.sub-menu:last").append('<li class="imgContent"><a href="http://nationalarchives.gov.uk/first-world-war/" title="First World War 100 - read about our centenary programme"><img src="http://www.nationalarchives.gov.uk/images/home/menu-first-world-war-b.jpg" alt="Explore First World War 100" class="img-responsive"></a></li>');
```

Comment:
Might be good to test that the injected image has an 'img-responsive' class applied.